modules/create-wif-cluster-cli.adoc

```diff
@@ -205,18 +221,31 @@ ocm gcp update wif-config --version <version> \ <1>
 <1> Replace `<version>` with the {product-title} y-stream version you plan to update the cluster to.
 <2> Replace `<wif_name>` with the name of the WIF configuration you want to update.
 
-[id="ocm-cli-list-wif-commands_{context}"]
-== List WIF clusters
+[id="ocm-cli-verify-wif-commands_{context}"]
+== Verifying a WIF configuration
+You can verify that the configuration of resources associated with a wif-config are correct by running the `ocm gcp verify wif-config` command. If a misconfiguration is found, output will identify the specific misconfiguration and recommend that you update the WIF configuration.
 
-To list all of your {product-title} clusters that have been deployed using the WIF authentication type, run the following command:
+You will need the name or ID of the wif-config you want to verify before verification.
+To obtain the name and ID of your active wif-configs, run the following command:
 
 [source,terminal]
 ----
-$ ocm list clusters --parameter search="gcp.authentication.wif_config_id != ''"
+$ ocm gcp list wif-configs
 ----
-To list all of your {product-title} clusters that have been deployed using a specific wif-config, run the following command:
+
+To determine if the wif-config you want to verify is configured correctly, run the following command, replacing `<wif_config_name>` or `<wif_config_id>` with the name or ID of the wif-config you want to verify:
+
 [source,terminal]
 ----
-$ ocm list clusters --parameter search="gcp.authentication.wif_config_id = '<wif_config_id>'" <1>
+$ ocm gcp verify wif-config <wif_config_name>|<wif_config_id>
 ----
-<1> Replace `<wif_config_id>` with the ID of the WIF configuration to list the clusters that have been deployed using that WIF configuration.
\ No newline at end of file
+--
+.Example output
+[source,terminal]
+----
+Error: verification failed with error: missing role 'compute.storageAdmin'.
+Running 'ocm gcp update wif-config' may fix errors related to cloud resource misconfiguration.
+exit status 1.
+----
+--
+In the example output displayed above, the specific misconfiguration is provided, along with the recommendation to update the wif-config as a possible solution.
```

Comment:
IMO, this info redundant as it's already mentioned before. I suggest to remove this sentence. But if you want to keep it, then please see the following suggested changes:

```suggestion
The example output displays the specific misconfiguration and the recommended solution to fix the error.
```